src/main/java/com/sngular/kloadgen/extractor/impl/SchemaExtractorImpl.java

```diff
@@ -64,7 +66,7 @@ public final Pair<String, List<FieldValueMapping>> flatPropertiesList(final Stri
 
     final var schemaParsed = JMeterHelper.getParsedSchema(subjectName, properties);
     final List<FieldValueMapping> attributeList = new ArrayList<>();
-
+final HashMap<String, TypeElement> nestedTypes = new HashMap<>();
```

Comment:
```suggestion
    final HashMap<String, TypeElement> nestedTypes = new HashMap<>();
```
This line wasn't indented